openmc/deplete/pool.py

```diff
@@ -10,10 +10,17 @@
 
 from openmc.mpi import comm
 
+import sys
+
 # Configurable switch that enables / disables the use of
 # multiprocessing routines during depletion
 USE_MULTIPROCESSING = True
 
+# Not sure why, but using multiprocessing on Windows leads to many transport
+# simulations being run over eachother and leads to catastrophe.
+if sys.platform == 'win32':
+    USE_MULTIPROCESSING = False
```

Comment:
I completely agree, though I have not had time to look at it since. It was a while ago when I examined this, but from what I could tell I think it is a Python/Windows problem and not necessarily a bug on our side.

Reply:
Just as a note, turns out this same problem exists on my student's M1 mac.